src/libraries/Common/src/Interop/Browser/Interop.JavaScript.ArrayBuffer.cs

```diff
@@ -0,0 +1,29 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+internal static partial class Interop
+{
+    internal static partial class JavaScript
+    {
+        public class ArrayBuffer : CoreObject
+        {
```

Comment:
Nit: Extra new line (not commenting on all places)

Reply:
Addressed in recent commit